src/pages/home/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Banner from './banner'
+
+const HomePage = () => {
+  return (
+    <div>
+      <h1>Home Page</h1>
+      <Banner />
```

Comment:
Splitting sections into different files should help us prevent a lot of merges conflicts

Reply:
@rvente @xuejingao @muskankapoor